content/themis/languages/python/installation.md

```diff
@@ -75,13 +75,151 @@ Once PyThemis is installed, you can [try out code examples](../examples/).
 If the stable package version does not suit your needs,
 you can manually build and install the latest version of Themis from source code.
 
+ 1. [Build and install Themis Core library](/themis/installation/installation-from-sources/)
+    into your system.
+
+ 2. Install Python dependencies
+
+    Depending on how you want to install PyThemis, there are different requirements.
+
+    To create a `.whl` package for installation inside virtualenv, you'll need `setuptools` and `wheel`
+
+    * Install globally
+
+      For Debian, Ubuntu:
+
+      ```bash
+      sudo apt install python3-setuptools python3-wheel
+      ```
+
+<!--
+      For CentOS, RHEL:
+
+      ```bash
+      sudo yum install python3-setuptools python3-wheel
+      ```
+-->
+
+    * Or install into virtualenv
+
+      ```bash
+      # activate virtualenv if not done yet
+      pip install setuptools wheel
+      ```
+
+    ---
+
+    To create a system package (i.e. `.deb` one for Debian) you will need `pip` and `fpm`
+
+    * Install globally
+
+      For Debian, Ubuntu:
+
+      ```bash
+      sudo apt install python3-pip lsb-release ruby
+      sudo gem install fpm
+      ```
+
+<!--
+      For CentOS, RHEL:
+
+      ```bash
+      sudo yum install python3-pip ruby
+      sudo gem install fpm
+      ```
+
+    On RHEL you also need to have working `pip` command. If it's not, and you only got `pip3`,
+    create a symlink like this: `sudo ln -s $(which pip3) /usr/bin/pip`.
+-->
+
+ 3. Create a PyThemis package
+
+    * Create a "wheel" for virtualenv
```

Comment:
I prefer to explicitly note for which way this case. For example `To install locally: Create a "wheel"...`. Or mentions at start that we will describe 2 ways, first for local installation and second one for system. And then mention it like **Way 1. ....**, **Way 2. ....**

Because when I read step 3, I didn't understand why we need .whl for system package below and went to the makefile in the themis...